services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/directory/decorator/DateAggregatingDecorator.java

```diff
@@ -93,8 +93,11 @@ public void prepare(ImmutableStack<Object> indices) {
               return aggregate;
             })
             .map(file -> new DiagnosisKeySigningDecorator(file, cryptoProvider, distributionServiceConfig))
-            .peek(currentDirectory::addWritable)
-            .forEach(aggregate -> aggregate.prepare(indices)));
+            .forEach(aggregate -> {
```

Comment:
@pithumke can you review this part?

Reply:
This functionally identical, but gets rid of a sonar warning, so 👍